tests/test_perceptual_loss.py

```diff
@@ -40,11 +40,26 @@
         (2, 1, 64, 64, 64),
         (2, 1, 64, 64, 64),
     ],
+    [
+        {"spatial_dims": 3, "network_type": "medicalnet_resnet10_23datasets", "is_fake_3d": False},
+        (2, 6, 64, 64, 64),
+        (2, 6, 64, 64, 64),
+    ],
+    [
+        {"spatial_dims": 3, "network_type": "medicalnet_resnet10_23datasets", "is_fake_3d": False, "channelwise": True},
```

Comment:
Could you please review all the instances where "channelwise" is used? It appears there may be a misnaming issue, even with the wrong name, it doesn't affect the passing of the CI tests.